vulkaninfo/vulkaninfo.h

```diff
@@ -243,7 +243,7 @@ auto GetVectorInit(const char *func_name, F &&f, T init, Ts &&...ts) -> std::vec
         err = f(ts..., &count, results.data());
         results.resize(count);
         iteration_count++;
-    } while (err == VK_INCOMPLETE || iteration_count < max_iterations);
+    } while (err == VK_INCOMPLETE || iteration_count >= max_iterations);
```

Comment:
What is the reason for this change? It looks wrong, could lead to infinite loop

Reply:
The reason for the change was that the loop was always run 3 times. I want the loop to only run once if necessary, but I can see how it would lead to infinite loops now.

I'll have to rethink the logic some more.